bodyfetcher.py

```diff
@@ -46,7 +46,6 @@ class BodyFetcher:
     api_data_lock = threading.Lock()
     queue_modify_lock = threading.Lock()
     max_ids_modify_lock = threading.Lock()
-    queue_timing_modify_lock = threading.Lock()
```

Comment:
I think we still want to keep the lock. AFAIK writing to a file is **not** atomic, so if two threads try to add timing data simultaneously we could have some wonderful and fun concurrency problems.